site/src/api.njk

```diff
@@ -173,7 +180,7 @@ Returns: `Promise`
 
 One or many URLs to be prerendered.
 
-> **Note:** As prerendering using Speculative Rules API only supports same-origin at this point, only same-origin urls are accepted. Any non same-origin urls will return a rejected Promise.
+> **Note:** Speculative Rules API supports same-site cross origin Prerendering with [opt-in header](https://docs.google.com/document/d/1WsDYA8NMCSwsK8dXCKdajdAd3ZcQUu9w1eoe0hEB_nU/).
```

Comment:
Minor: it would be nice if we could put this behind a URL shortener vs. including a completely Google Docs URL in source. I created https://bit.ly/ss-cross-origin-pre for this purpose if you'd like to use that.

Reply:
Thanks for the review, suggestion and helping to create the short URL. It has been updated in the latest minor commit.